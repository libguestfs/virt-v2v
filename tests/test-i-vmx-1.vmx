.encoding = "UTF-8"
config.version = "8"
virtualHW.version = "8"
nvram = "BZ1308535_21disks.nvram"
pciBridge0.present = "TRUE"
svga.present = "TRUE"
pciBridge4.present = "TRUE"
pciBridge4.virtualDev = "pcieRootPort"
pciBridge4.functions = "8"
pciBridge5.present = "TRUE"
pciBridge5.virtualDev = "pcieRootPort"
pciBridge5.functions = "8"
pciBridge6.present = "TRUE"
pciBridge6.virtualDev = "pcieRootPort"
pciBridge6.functions = "8"
pciBridge7.present = "TRUE"
pciBridge7.virtualDev = "pcieRootPort"
pciBridge7.functions = "8"
vmci0.present = "TRUE"
hpet0.present = "TRUE"
displayName = "BZ1308535_21disks"
extendedConfigFile = "BZ1308535_21disks.vmxf"
virtualHW.productCompatibility = "hosted"
memSize = "2048"
sched.cpu.units = "mhz"
powerType.powerOff = "soft"
powerType.suspend = "hard"
powerType.reset = "soft"
scsi0.virtualDev = "pvscsi"
scsi0.present = "TRUE"
scsi1.virtualDev = "pvscsi"
scsi1.present = "TRUE"
ide1:0.deviceType = "cdrom-image"
ide1:0.fileName = "/vmfs/volumes/5458b680-34ec3500-9f36-001320f5f6ca/ISOs/RHEL-7.1-20150219.1-Server-x86_64-boot.iso"
ide1:0.present = "TRUE"
floppy0.startConnected = "FALSE"
floppy0.clientDevice = "TRUE"
floppy0.fileName = "vmware-null-remote-floppy"
ethernet0.virtualDev = "vmxnet3"
ethernet0.networkName = "VM Network"
ethernet0.addressType = "generated"
ethernet0.present = "TRUE"
scsi0:0.deviceType = "scsi-hardDisk"
scsi0:0.fileName = "BZ1308535_21disks.vmdk"
scsi0:0.present = "TRUE"
scsi0:1.deviceType = "scsi-hardDisk"
scsi0:1.fileName = "BZ1308535_21disks_1.vmdk"
scsi0:1.present = "TRUE"
scsi0:2.deviceType = "scsi-hardDisk"
scsi0:2.fileName = "BZ1308535_21disks_2.vmdk"
scsi0:2.present = "TRUE"
scsi0:3.deviceType = "scsi-hardDisk"
scsi0:3.fileName = "BZ1308535_21disks_3.vmdk"
scsi0:3.present = "TRUE"
scsi0:4.deviceType = "scsi-hardDisk"
scsi0:4.fileName = "BZ1308535_21disks_4.vmdk"
scsi0:4.present = "TRUE"
scsi0:5.deviceType = "scsi-hardDisk"
scsi0:5.fileName = "BZ1308535_21disks_5.vmdk"
scsi0:5.present = "TRUE"
scsi0:6.deviceType = "scsi-hardDisk"
scsi0:6.fileName = "BZ1308535_21disks_6.vmdk"
scsi0:6.present = "TRUE"
scsi0:8.deviceType = "scsi-hardDisk"
scsi0:8.fileName = "BZ1308535_21disks_7.vmdk"
scsi0:8.present = "TRUE"
scsi0:9.deviceType = "scsi-hardDisk"
scsi0:9.fileName = "BZ1308535_21disks_8.vmdk"
scsi0:9.present = "TRUE"
scsi0:10.deviceType = "scsi-hardDisk"
scsi0:10.fileName = "BZ1308535_21disks_9.vmdk"
scsi0:10.present = "TRUE"
scsi0:11.deviceType = "scsi-hardDisk"
scsi0:11.fileName = "BZ1308535_21disks_10.vmdk"
scsi0:11.present = "TRUE"
scsi0:12.deviceType = "scsi-hardDisk"
scsi0:12.fileName = "BZ1308535_21disks_11.vmdk"
scsi0:12.present = "TRUE"
scsi0:13.deviceType = "scsi-hardDisk"
scsi0:13.fileName = "BZ1308535_21disks_12.vmdk"
scsi0:13.present = "TRUE"
scsi0:14.deviceType = "scsi-hardDisk"
scsi0:14.fileName = "BZ1308535_21disks_13.vmdk"
scsi0:14.present = "TRUE"
scsi0:15.deviceType = "scsi-hardDisk"
scsi0:15.fileName = "BZ1308535_21disks_14.vmdk"
scsi0:15.present = "TRUE"
scsi1:0.deviceType = "scsi-hardDisk"
scsi1:0.fileName = "BZ1308535_21disks_15.vmdk"
scsi1:0.present = "TRUE"
scsi1:1.deviceType = "scsi-hardDisk"
scsi1:1.fileName = "BZ1308535_21disks_16.vmdk"
scsi1:1.present = "TRUE"
scsi1:2.deviceType = "scsi-hardDisk"
scsi1:2.fileName = "BZ1308535_21disks_17.vmdk"
scsi1:2.present = "TRUE"
scsi1:3.deviceType = "scsi-hardDisk"
scsi1:3.fileName = "BZ1308535_21disks_18.vmdk"
scsi1:3.present = "TRUE"
scsi1:4.deviceType = "scsi-hardDisk"
scsi1:4.fileName = "BZ1308535_21disks_19.vmdk"
scsi1:4.present = "TRUE"
scsi1:5.deviceType = "scsi-hardDisk"
scsi1:5.fileName = "BZ1308535_21disks_20.vmdk"
scsi1:5.present = "TRUE"
guestOS = "rhel6-64"
toolScripts.afterPowerOn = "TRUE"
toolScripts.afterResume = "TRUE"
toolScripts.beforeSuspend = "TRUE"
toolScripts.beforePowerOff = "TRUE"
uuid.bios = "56 4d 96 af e6 46 bd 86-5c 4d 65 4e 77 36 ef 31"
uuid.location = "56 4d 96 af e6 46 bd 86-5c 4d 65 4e 77 36 ef 31"
vc.uuid = "52 31 cb fc c1 3f 96 32-83 c0 bb 70 6c 90 5c fd"
chipset.onlineStandby = "FALSE"
sched.cpu.min = "0"
sched.cpu.shares = "normal"
sched.mem.min = "0"
sched.mem.minSize = "0"
sched.mem.shares = "normal"
svga.vramSize = "8388608"
sched.swap.derivedName = "/vmfs/volumes/5458b680-34ec3500-9f36-001320f5f6ca/BZ1308535_21disks/BZ1308535_21disks-6a024f8a.vswp"
replay.supported = "FALSE"
replay.filename = ""
scsi0:0.redo = ""
scsi0:1.redo = ""
scsi0:2.redo = ""
scsi0:3.redo = ""
scsi0:4.redo = ""
scsi0:5.redo = ""
scsi0:6.redo = ""
scsi0:8.redo = ""
scsi0:9.redo = ""
scsi0:10.redo = ""
scsi0:11.redo = ""
scsi0:12.redo = ""
scsi0:13.redo = ""
scsi0:14.redo = ""
scsi0:15.redo = ""
scsi1:0.redo = ""
scsi1:1.redo = ""
scsi1:2.redo = ""
scsi1:3.redo = ""
scsi1:4.redo = ""
scsi1:5.redo = ""
pciBridge0.pciSlotNumber = "17"
pciBridge4.pciSlotNumber = "21"
pciBridge5.pciSlotNumber = "22"
pciBridge6.pciSlotNumber = "23"
pciBridge7.pciSlotNumber = "24"
scsi0.pciSlotNumber = "160"
scsi1.pciSlotNumber = "192"
ethernet0.pciSlotNumber = "224"
vmci0.pciSlotNumber = "32"
scsi0.sasWWID = "50 05 05 6f e6 46 bd 80"
scsi1.sasWWID = "50 05 05 6f e6 46 bc 80"
ethernet0.generatedAddress = "00:0c:29:36:ef:31"
ethernet0.generatedAddressOffset = "0"
vmci0.id = "2000088881"
hostCPUID.0 = "0000000d756e65476c65746e49656e69"
hostCPUID.1 = "000206a700100800179ae3bfbfebfbff"
hostCPUID.80000001 = "00000000000000000000000128100800"
guestCPUID.0 = "0000000d756e65476c65746e49656e69"
guestCPUID.1 = "000206a700010800969822030fabfbff"
guestCPUID.80000001 = "00000000000000000000000128100800"
userCPUID.0 = "0000000d756e65476c65746e49656e69"
userCPUID.1 = "000206a700100800169822030fabfbff"
userCPUID.80000001 = "00000000000000000000000128100800"
evcCompatibilityMode = "FALSE"
vmotion.checkpointFBSize = "8388608"
cleanShutdown = "TRUE"
softPowerOff = "TRUE"
tools.remindInstall = "TRUE"
