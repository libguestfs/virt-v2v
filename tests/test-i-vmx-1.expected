
Source guest information (--print-source option):

    source name: BZ1308535_21disks
hypervisor type: vmware
       VM genid: 
         memory: 2147483648 (bytes)
       nr vCPUs: 1
     CPU vendor: 
      CPU model: 
   CPU topology: 
   CPU features: 
       firmware: bios
        display: 
          sound: 
disks:
	0 [scsi]
	1 [scsi]
	2 [scsi]
	3 [scsi]
	4 [scsi]
	5 [scsi]
	6 [scsi]
	7 [scsi]
	8 [scsi]
	9 [scsi]
	10 [scsi]
	11 [scsi]
	12 [scsi]
	13 [scsi]
	14 [scsi]
	15 [scsi]
	16 [scsi]
	17 [scsi]
	18 [scsi]
	19 [scsi]
	20 [scsi]
removable media:
	CD-ROM [ide] in slot 2
NICs:
	Network "VM Network" mac: 00:0c:29:36:ef:31 [vmxnet3]

